tests/e2e.py

```diff
@@ -4,43 +4,31 @@
 from selenium.webdriver.common.keys import Keys
 from trello import TrelloClient
 import os
-import requests
+import pymongo
 import pytest
 import app
+import random
+import string
 
-TRELLO_URL = "https://api.trello.com/1"
+def get_db_collection():
 
-def create_trello_board():
-    response = requests.post(
-        url=f'{TRELLO_URL}/boards',
-        params={
-            'key': os.getenv("API_KEY"),
-            'token': os.getenv("TOKEN"),
-            'name': 'Selenium Test Board',
-        }
-    )
-    return response.json()['id']
+    dbClientUri = os.getenv('MONGO_DB_CONNECTION')
+    databaseName = os.getenv('MONGO_DB_DATABASE')
+    collectionName = 'collection'
 
-def delete_trello_board(board_id):
-    requests.delete(
-        url=f'{TRELLO_URL}/boards/{board_id}',
-        params={
-            'key': os.getenv("API_KEY"),
-            'token': os.getenv("TOKEN"),
-        }
-    )
+    dbClient = pymongo.MongoClient(dbClientUri)
+    db = dbClient[databaseName]
+    return db[collectionName]
     
 @pytest.fixture(scope='module')
 def test_app():
     file_path = find_dotenv('.env')
     load_dotenv(file_path, override=True)
 
-    # Create the new board & update the board id environment variable
-    board_id = create_trello_board()
-    os.environ['BOARD_ID'] = board_id
-    board_list = TrelloClient(os.getenv("API_KEY"),os.getenv("TOKEN")).get_board_lists(board_id)
-
-    os.environ['TODO_LIST_ID'] = board_list[0]['id']
+    os.environ['MONGO_DB_DATABASE_NAME'] = 'test-' + \
```

Comment:
This test looks like it should be finished & working but for the fact that you have mismatched this variable name - if this was `MONGO_DB_DATABASE` then your app would run against your test database rather than your live database
Once this is working, don't forget to undo the rename so the test doesn't get missed